### DISCUSSION TITLE
Caliburn.Micro V4.0.210 Released

### DISCUSSION THREAD

**Discussion by vb2ae** [Caliburn-Micro/Caliburn.Micro#812] [Announcements]
Caliburn.Micro V4.0.210 Released

.net 6 support
moved min version of .net framework to 4.6.2
Check for multiple assemblies being added to the cache
IOC performance fixes

**Comment by Jankiasati** [comment]
Hi, Will it support 4.8.1 dot net framework

**Reply by vb2ae** [reply]
Yes
